content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -15,9 +15,21 @@ Refer to the [prerequisites](/docs/setup/kubernetes/quick-start/#prerequisites)
 
 ## Installation steps
 
-You have two mutually exclusive options to install Istio:
-- To use Kubernetes manifests to deploy Istio, follow the instructions for [option #1](#option-1).
-- To use [Helm's Tiller pod](https://helm.sh/) to manage your Istio release, follow the instructions for [option #2](#option-2).
+The following commands have relative references in the Istio directory. You must execute the commands in Istio's root directory.
```

Comment:
Consider adding this note under prerequisites.

Reply:
moved!